marimo/_plugins/core/web_component.py

```diff
@@ -22,27 +27,26 @@
 
     from typing import Optional
 
-from marimo._output.md import _md
-from marimo._output.mime import MIME
-from marimo._plugins.core.json_encoder import WebComponentEncoder
-
-JSONType: TypeAlias = Union[
-    Mapping[str, "JSONType"],
-    Sequence["JSONType"],
-    str,
-    int,
-    float,
-    bool,
-    object,
-    MIME,  # MIME is a JSONType since we have a custom JSONEncoder for it
-    None,
-]
+    JSONType: TypeAlias = Union[
+        Mapping[str, "JSONType"],
+        Sequence["JSONType"],
+        str,
+        int,
+        float,
+        bool,
+        object,
+        MIME,  # MIME is a JSONType since we have a custom JSONEncoder for it
+        None,
+    ]
+
+else:
+    JSONType = Any
 
 S = TypeVar("S", bound=JSONType)
 
 
 def _build_attr(name: str, value: JSONType) -> str:
-    processed = escape(json.dumps(value, cls=WebComponentEncoder))
+    processed = escape(encoder.encode(value).decode("utf-8"))
```

Comment:
could we expose some utils from this module:

`def to_json_str()`
`def to_dict()`

Reply:
good idea. I added `encode_json_str`, `encode_json_bytes`, and `asdict`.